main.nf

```diff
@@ -120,6 +120,10 @@ ch_biotypes_header = Channel.fromPath("$baseDir/assets/biotypes_header.txt", che
 Channel.fromPath("$baseDir/assets/where_are_my_files.txt", checkIfExists: true)
        .into{ch_where_trim_galore; ch_where_star; ch_where_hisat2; ch_where_hisat2_sort}
 
+if (params.gencode) {
```

Comment:
It would be better to put this in the `featureCounts` process directly using an evaluation:
`def biotype = params.gencode ? 'gene_type' : params.fc_group_features_type`

Reply:
No, its just because its easier to read the code if we put it in the `process` in which it is evaluated - as opposed to scrolling much higher up in the script.

I tend to put all parameters in the `summary` section. If I parameter has a default value thats not boolean then I print that, and if its set to `false` by default I use a conditional statement to print it:
https://github.com/nf-core/chipseq/blob/333bba334512625775157aa0087c023cf0bde397/main.nf#L228-L246

But yes, I would put it in the summary section too 👍